physics/GFS_rrtmgp_thompsonmp_pre.F90

```diff
@@ -0,0 +1,297 @@
+! ########################################################################################
+! This module contains the interface between the THOMPSON macrophysics and the RRTMGP radiation
+! schemes. Only compatable with Model%imp_physics = Model%imp_physics_thompson
+! ########################################################################################
+module GFS_rrtmgp_thompsonmp_pre
+  use machine, only: &
+       kind_phys
+  use radiation_tools, only: &
+       check_error_msg
+  use module_mp_thompson, only: &
+       calc_effectRad, Nt_c,    &
+       re_qc_min, re_qc_max,    &
+       re_qi_min, re_qi_max,    &
+       re_qs_min, re_qs_max
+  use module_mp_thompson_make_number_concentrations, only: &
+       make_IceNumber,      &
+       make_DropletNumber, &
+       make_RainNumber
+  use rrtmgp_lw_cloud_optics, only: radliq_lwr => radliq_lwrLW, radliq_upr => radliq_uprLW,&
+                                    radice_lwr => radice_lwrLW, radice_upr => radice_uprLW
+  implicit none
+
+  ! Parameters specific to THOMPSON MP scheme.
+  real(kind_phys), parameter :: &
+       rerain_def = 1000.0 ! Default rain radius to 1000 microns
+
+  public GFS_rrtmgp_thompsonmp_pre_init, GFS_rrtmgp_thompsonmp_pre_run, GFS_rrtmgp_thompsonmp_pre_finalize
```

Comment:
See comment below.

Reply:
This file is not used. I will delete this file.